conf/machine/include/imx-base.inc

```diff
@@ -4,6 +4,7 @@ include conf/machine/include/fsl-default-settings.inc
 include conf/machine/include/fsl-default-versions.inc
 
 require conf/machine/include/utilities.inc
+require conf/machine/include/imx-mfgtoolconfig.inc
```

Comment:
Please drop this commit so we can merge it.

Reply:
Done @otavio